test/pay/subscription_renewing_mail_test.rb

```diff
@@ -0,0 +1,30 @@
+require "test_helper"
+
+class SubscriptionRenewingMailTest < ActiveSupport::TestCase
+  include ActionMailer::TestHelper
+
+  test "monthly subscription should receive correct renewal email" do
+    subscription = pay_subscriptions(:full_time)
+    event = new_upcoming_invoice_event(subscription)
+
+    assert_emails 1 do
```

Comment:
Let's use `assert_enqueued_email_with` so we can assert the correct mail and args.

Example on `test/integration/messages_test.rb:131`.

Reply:
Changed 👍